translating-guide/en/customize-locale.md

```diff
@@ -0,0 +1,96 @@
+# Customize a Translation Locally
+
+You can use the Custom Locale plugin to do a translation or change a translation on your local installation. It also allows you to customize text on your site in any language that is installed and enabled. For example, if you want to change the text displayed on the review recommendation buttons in the submission workflow, you can use the Custom Locale plugin. 
+
+The Custom Locale is only reliably available and operational for OJS/OMP/OPS 3.2.0+. 
+
+If you want to make a translation that other users can benefit from, it is better to translate in Weblate. But there are some situations in which translating on your local installation is preferred, and the Custom Locale plugin allows you to do this. This chapter explains how to use the plugin.
+
+Please see [How Languages and Locales Work](https://docs.pkp.sfu.ca/translating-guide/en/managing-languages#how-languages-and-locales-work) in this guide for an explanation of how languages and locale files work in PKP software. 
+
+## Install and Enable the Custom Locale Plugin
+
+First, ensure that the plugin is installed and enable it:
+1. Go to Settings > Website > Plugins
+2. Find the Custom Locale Plugin in the list of Installed Plugins (if it is not listed with the Installed Plugins, check the Plugin Gallery and you may need to install it first)
+3. Click the box to enable it
+4. Click the blue arrow next to the plugin name and then the *Customize* link that appears below it. This will refresh the page and now you should see a new tab in the Website Settings menu called *Locales*.
+
+## How the Custom Locale Plugin Works
+
+Under the Locales tab you will see a list of locale files that contain all of the message keys that appear as text on your site, for the locale you have set as the Primary Locale under Website Settings > Setup > Languages. So, if you have more than one language on your site but Português (Brasil) is the Primary Locale, the locale files you see in the Locales tab will be for the Português (Brasil) language.  
+
+![](./assets/translating-guide-custom-locale-localefiles.png)
+
+If you want to edit the locale files for a different language installed on your site, you will need to temporarily set it as the Primary Locale and then change the Primary Locale when you’re finished editing the locales.
```

Comment:
This really and needs to be fixed in the plugin! But OK, let's document it here for now. I've filed the problem at https://github.com/pkp/customLocale/issues/8 and maybe we should add a link to the issue from here as a to-be-fixed?

Reply:
I generally don't like referring to Github issues in documentation but when this is fixed I can update the documentation.

Reply:
Agreed, and fundamentally I'd rather fix the issue than link to it, but in cases like this where things are obviously weird and bad I think it's nice to indicate at least that we're aware.